### ISSUE TITLE
follow setDeprecated signature of Symfony 5.1

### ISSUE THREAD

**Issue by finnef** [dustin10/VichUploaderBundle#1134]
### Bug Report

|    Q            |   A
|---------------- | ------
| BC Break        | no
| Bundle version  | 1.13.2
| Symfony version | 5.1.0
| PHP version     | 7.3.9

#### Summary

When upgrading to symfony 5.1 a deprecation notice is generated about setDeprecated()

#### Current behavior

```Since symfony/config 5.1: The signature of method "Symfony\Component\Config\Definition\Builder\NodeDefinition::setDeprecated()" requires 3 arguments: "string $package, string $version, string $message", not defining them is deprecated.
Hide context    Show trace

[▼
  "exception" => Symfony\Component\ErrorHandler\Exception\SilencedErrorContext {#3661 ▼
    +count: 2
    -severity: E_USER_DEPRECATED
    trace: {▼
      /Applications/MAMP/htdocs/csr-cs/vendor/symfony/config/Definition/Builder/NodeDefinition.php:176 {▶}
      /Applications/MAMP/htdocs/csr-cs/vendor/vich/uploader-bundle/src/DependencyInjection/Configuration.php:60 {▶}
    }
  }
]```

#### How to reproduce

This was generated on every request in an app that has a file entity which is 'vich uploadable' annotated, even if the page had nothing to do with the file entity.

#### Expected behavior

Not generate a deprecated notice. It worked fine in Symfony 5.0

#### Proposed fix

Change the call to setDeprecated() in src/DependencyInjection/Configuration.php:60 to include all the required parameters.

**Comment by garak** [comment]
This was fixed in #1132
You can use dev-master or (even better) 1.14@dev.
Or, of course, you can wait for next release (it should be in a week or so)

**Comment by finnef** [comment]
👍
